assets/js/src/newsletter_editor/components/history.js

```diff
@@ -62,12 +62,18 @@ Module.HistoryView = Marionette.View.extend({
     if (!this.canUndo()) {
       return;
     }
+    this.model.currentStateIndex += 1;
```

Comment:
Could it occur by some mistake you'd get out of the array bounds with this index? And for "redo" below 0? Should we check/sanitize it here (i.e. with min/max functions)?